docs/installation.rst

```diff
@@ -60,3 +60,10 @@ After downloading, make sure you include the FFMPEG binary directory (e.g. C:\\u
 
 On Windows you can edit your environment variables and edit your Path to add to it. You may want to restart your PC after doing so too.
 
+Paltas
+------
```

Comment:
do you also want to list how to actually install plates, i.e. $pip install paltas
(perhaps even state the version this code has run (who knows, perhaps a later version would require changes)